apps/web/src/components/Shared/Follow.tsx

```diff
@@ -132,6 +134,7 @@ const Follow: FC<Props> = ({ profile, showText = false, setFollowing, followSour
 
   const createFollow = async () => {
     if (!currentProfile) {
+      openConnectModal?.();
```

Comment:
@bigint  Now we need to call both **openConnectModal** and **setShowAuthModal**.
So we might want to move them to a custom hook.

```
openConnectModal?.();
setShowAuthModal(true);
```